src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/FileUpload/BitFileUploadDemo.razor.cs

```diff
@@ -237,34 +237,40 @@ string GetBaseUrl()
             }
         };
 
-        private readonly string example1HTMLCode = @"<BitFileUpload Label=""Select or drag and drop files""
-               UploadUrl=""@UploadUrl""
-               RemoveUrl=""@RemoveUrl"">
-</BitFileUpload>";
+        private const string Example1CSharpCode = @"
```

Comment:
plz don't change any code style like this one in this PR.
so plz revert these renames too. changing them to `const` requires some other renames which is not part of this PR.